content/en/blog/_posts/2017-02-00-Postgresql-Clusters-Kubernetes-Statefulsets.md

```diff
@@ -41,7 +41,7 @@ sudo chmod 777 /nfsfileshare/
 
 sudo vi /etc/exports
 
-sudo exportfs -r
+sudo exports -r
```

Comment:
This is wrong. exportfs is the correct command, see "man exportfs"

Reply:
Great catch @ajaeger! @angao please push another commit that reverts this.